src/reference/antora/modules/ROOT/pages/amqp/connections.adoc

```diff
@@ -28,6 +28,8 @@ Simple publisher confirmations are supported by all three factories.
 When configuring a `RabbitTemplate` to use a xref:amqp/template.adoc#separate-connection[separate connection], you can now, starting with version 2.3.2, configure the publishing connection factory to be a different type.
 By default, the publishing factory is the same type and any properties set on the main factory are also propagated to the publishing factory.
 
+Starting with version 3.1, the connectionCreatingBackOff property is supported to define backoff policy when creating connection channel.
```

Comment:
The `connectionCreatingBackOff` is a code, therefore has to be rendered respectively.
And not clear from this sentence what this property belongs to.
Would you mind be more specific and really explain the reasoning behind this property?
In this sentence I mean.
Make it really useful for the community!

Reply:
I am very willing to let the community know about the enhanced uses. There might have been a misunderstanding in my language expression, so I have made a revision. Please confirm it again, thank you.